src/main/java/core/basesyntax/impl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileReaderServiceImpl implements FileReaderService {
```

Comment:
```suggestion
public class FileReaderServiceImpl implements FileReaderService {
```
we can read not only from csv